src/main/java/pl/cinema/server/reservation/Maker.java

```diff
@@ -0,0 +1,20 @@
+package pl.cinema.server.reservation;
+
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "customers")
+class Maker {
```

Comment:
Co ta klasa reprezentuje? Z nazwy, nawet przetlumaczonej na polski (`robiacz`?) niewiele wnioskuję.

Reply:
ogólnie jest w odpowiednim pakiecie ta klasa więc wiadomo z kontekstu pakietu w jakim jesteśmy, że jest to osoba robiąca rezerwację - package reservation. czasem pomijam duplikację nazewnictwa z pakietu dla uproszczenia

Reply:
jest używana w Reservation do mapowania, mamy customer_id przy rezerwacji.